docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -122,245 +141,190 @@ RUN yum update -y && yum install -y https://dl.fedoraproject.org/pub/epel/epel-r
     yum remove -y epel-release
 ```
 
-or you can use the installation script `install_NEO_OCL_driver.sh` if you previously installed OpenVINO in the Dockerfile, where `INTEL_OPENCL` is the variable to store the default version of Intel® Graphics Compute Runtime for OpenCL™ Driver:
+or you can use the installation script `install_NEO_OCL_driver.sh` if you previously installed OpenVINO in the Dockerfile. The installation script will install the recommended version of GPU driver for the operating system in the image.
 
 ```sh
 WORKDIR /tmp/opencl
 RUN useradd -ms /bin/bash -G video,users openvino && \
     chown openvino -R /home/openvino
 RUN groupmod -g 44 video
 
-# Please use `20.35.17767` for 10th generation Intel® Core™ processor (formerly Ice Lake) or 11th generation Intel® Core™ processor (formerly Tiger Lake)
-ARG INTEL_OPENCL=19.41.14441
-
 WORKDIR ${INTEL_OPENVINO_DIR}/install_dependencies
-RUN ./install_NEO_OCL_driver.sh --no_numa -y --install_driver ${INTEL_OPENCL} && \
+RUN ./install_NEO_OCL_driver.sh --no_numa -y  && \
     yum clean all && rm -rf /var/cache/yum && \
     yum remove -y epel-release
 ```
```

Comment:
We can remove this because these lines are already included in OpenVINO dockerfiles and we are already distributing the images with the recommended version of OpenCL runtime included

Reply:
OK, updated as suggested.